libraries/botbuilder/tests/cloudAdapter.test.js

```diff
@@ -101,11 +101,11 @@ describe('CloudAdapter', function () {
         });
 
         it('throws exception on expired token', async function () {
-            const consoleSpy = sinon.spy(console, 'error');
+            const consoleStub = sinon.stub(console, 'error');
```

Comment:
There's a beforeEach that creates a sandbox for sinon. We can use that one in each test.
```suggestion
            const consoleStub = sandbox.stub(console, 'error');
```